### PR TITLE
Adds verbose logging to the emergency shuttle console

### PR DESCRIPTION
**Pull request by bloons3**
## About The Pull Request

Adds a specific log for when authorization to early launch is revoked, or when auth is removed.

## Why It's Good For The Game

Admins can now see when the HoP cancels the emergency launch because they left Ian behind in their office.

Will add: 
https://github.com/tgstation/tgstation/pull/52522
when I can figure out how to cherry pick

## Changelog
:cl:
admin: increased emergency shuttle verbosity logging
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
Definitely include that port.

**Comment by bloons3** [comment]
I borked the format string, lemme double check locally

**Comment by bloons3** [comment]
image

```	Line 53: [2020-12-16 09:16:24.518] GAME: bloons3/(Ashlie Beedell) has authorized early shuttle launch in (205,145,2), 1 of 3 needed..
	Line 54: [2020-12-16 09:16:26.080] GAME: bloons3/(Ashlie Beedell) has deauthorized early shuttle launch in (205,145,2) 0 of 3 needed.
	Line 56: [2020-12-16 09:16:27.596] GAME: bloons3/(Ashlie Beedell) has deauthorized early shuttle launch in (205,145,2) 0 of 3 needed.
	Line 57: [2020-12-16 09:16:30.715] GAME: bloons3/(Ashlie Beedell) has authorized early shuttle launch in (205,145,2), 1 of 3 needed..
	Line 58: [2020-12-16 09:16:31.360] GAME: bloons3/(Ashlie Beedell) has revoked early shuttle launch in (205,145,2).
	Line 62: [2020-12-16 09:16:37.549] GAME: bloons3/(Ashlie Beedell) has authorized early shuttle launch in (205,145,2), 1 of 3 needed..
	Line 63: [2020-12-16 09:16:39.833] ADMIN: bloons3/(Ashlie Beedell) admin ghosted.
	Line 65: [2020-12-16 09:16:39.834] ACCESS: Mob Login: bloons3/(Ashlie Beedell) was assigned to a /mob/dead/observer
	Line 66: [2020-12-16 09:16:39.844] GAME: bloons3/(Ashlie Beedell) Client bloons3/(Ashlie Beedell) has taken ownership of mob Ashlie Beedell(/mob/dead/observer) (Emergency Shuttle (206, 144, 2))
	Line 66: [2020-12-16 09:16:39.844] GAME: bloons3/(Ashlie Beedell) Client bloons3/(Ashlie Beedell) has taken ownership of mob Ashlie Beedell(/mob/dead/observer) (Emergency Shuttle (206, 144, 2))
	Line 69: [2020-12-16 09:16:42.271] ADMIN: bloons3/(Ashlie Beedell) stuffed bloons3 into Beckett Rathens.
	Line 69: [2020-12-16 09:16:42.271] ADMIN: bloons3/(Ashlie Beedell) stuffed bloons3 into Beckett Rathens.
	Line 71: [2020-12-16 09:16:42.272] ACCESS: Mob Login: bloons3/(Beckett Rathens) was assigned to a /mob/living/carbon/human
	Line 72: [2020-12-16 09:16:42.275] GAME: bloons3/(Beckett Rathens) Client bloons3/(Beckett Rathens) has taken ownership of mob Beckett Rathens(/mob/living/carbon/human) (Emergency Shuttle (205, 143, 2))
	Line 72: [2020-12-16 09:16:42.275] GAME: bloons3/(Beckett Rathens) Client bloons3/(Beckett Rathens) has taken ownership of mob Beckett Rathens(/mob/living/carbon/human) (Emergency Shuttle (205, 143, 2))
	Line 74: [2020-12-16 09:16:57.534] ADMIN: bloons3/(Beckett Rathens) spawned /obj/item/card/id/captains_spare at Emergency Shuttle (205, 143, 2)
	Line 78: [2020-12-16 09:17:04.621] GAME: bloons3/(Beckett Rathens) has authorized early shuttle launch in (205,145,2), 2 of 3 needed..
	Line 79: [2020-12-16 09:17:07.602] GAME: bloons3/(Beckett Rathens) has deauthorized early shuttle launch in (205,145,2) 1 of 3 needed.
	Line 82: [2020-12-16 09:17:10.194] GAME: bloons3/(Beckett Rathens) has authorized early shuttle launch in (205,145,2), 2 of 3 needed..
	Line 84: [2020-12-16 09:17:12.202] ADMIN: bloons3/(Beckett Rathens) admin ghosted.
	Line 86: [2020-12-16 09:17:12.204] ACCESS: Mob Login: bloons3/(Beckett Rathens) was assigned to a /mob/dead/observer
	Line 87: [2020-12-16 09:17:12.213] GAME: bloons3/(Beckett Rathens) Client bloons3/(Beckett Rathens) has taken ownership of mob Beckett Rathens(/mob/dead/observer) (Emergency Shuttle (205, 144, 2))
	Line 87: [2020-12-16 09:17:12.213] GAME: bloons3/(Beckett Rathens) Client bloons3/(Beckett Rathens) has taken ownership of mob Beckett Rathens(/mob/dead/observer) (Emergency Shuttle (205, 144, 2))
	Line 89: [2020-12-16 09:17:14.198] ADMIN: bloons3/(Beckett Rathens) stuffed bloons3 into Simon Logue.
	Line 89: [2020-12-16 09:17:14.198] ADMIN: bloons3/(Beckett Rathens) stuffed bloons3 into Simon Logue.
	Line 91: [2020-12-16 09:17:14.199] ACCESS: Mob Login: bloons3/(Simon Logue) was assigned to a /mob/living/carbon/human
	Line 92: [2020-12-16 09:17:14.202] GAME: bloons3/(Simon Logue) Client bloons3/(Simon Logue) has taken ownership of mob Simon Logue(/mob/living/carbon/human) (Emergency Shuttle (204, 143, 2))
	Line 92: [2020-12-16 09:17:14.202] GAME: bloons3/(Simon Logue) Client bloons3/(Simon Logue) has taken ownership of mob Simon Logue(/mob/living/carbon/human) (Emergency Shuttle (204, 143, 2))
	Line 97: [2020-12-16 09:17:31.340] ADMIN: bloons3/(Simon Logue) changed the equipment of bloons3/(Simon Logue) to /datum/outfit/death_commando.
	Line 97: [2020-12-16 09:17:31.340] ADMIN: bloons3/(Simon Logue) changed the equipment of bloons3/(Simon Logue) to /datum/outfit/death_commando.
	Line 99: [2020-12-16 09:17:33.630] GAME: bloons3/(Simon Logue) has authorized early shuttle launch in (205,145,2), 3 of 3 needed..
```

File: code/modules/shuttle/emergency.dm [hunk 1]
```diff
@@ -83,6 +83,8 @@
 			. = authorize(user)
 
 		if("repeal")
+			if(!(ID in authorized))
```
Comment (bloons3) [inline code/modules/shuttle/emergency.dm:86]: Prevents retracting auth without having given it.
Stops possible log/admin spam
[2020-12-16 09:16:26.080] GAME: bloons3/(Ashlie Beedell) has deauthorized early shuttle launch in (205,145,2) 0 of 3 needed.
[2020-12-16 09:16:27.596] GAME: bloons3/(Ashlie Beedell) has deauthorized early shuttle launch in (205,145,2) 0 of 3 needed.

**Review by PowerfulBacon — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
probably should have read ikes thing saying to include the port

**Comment by bloons3** [comment]
I'll do it separately

**Comment by bloons3** [comment]
https://github.com/BeeStation/BeeStation-Hornet/pull/3175

**Pull request merged**